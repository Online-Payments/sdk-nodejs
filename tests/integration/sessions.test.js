const sdkInstance = require('./__fixture__');
const config = require('./__fixture__/config');
const stubs = require('./__stubs__');

describe('Sessions', () => {
  test('it can create a session', (done) => {
    sdkInstance.sessions.createSession(config.merchantId, stubs.createSession, {}, (error, sdkResponse) => {
      expect(error).toBeNull();
      expect(sdkResponse.isSuccess).toBe(true);
      expect(sdkResponse).toMatchSnapshot();
      done();
    });
  });
});
