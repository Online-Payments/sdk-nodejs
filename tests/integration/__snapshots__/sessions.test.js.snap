// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sessions it can create a session 1`] = `
Object {
  "body": Object {
    "assetUrl": "string",
    "clientApiUrl": "string",
    "clientSessionId": "string",
    "customerId": "string",
    "invalidTokens": "['0a7042d6-bf24-40b6-85c5-dec24b9dd1d1', 'b7042dff-1fbd-46d2-9ecb-bddaa2c31165']",
  },
  "isSuccess": true,
  "status": 201,
}
`;
